omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -58,6 +59,31 @@ def solr_root
     host
   end
 
+  def elastic_search_major_version
```

Comment:
I'm a bit concerned that a transient ES issue during `reconfigure` could rather silently (one log message in hundreds of lines of output) reconfigure us to use version 2 instead of version 5 and produce very confusing errors for the user.